docs/cli.rst

```diff
@@ -0,0 +1,853 @@
+
+Usage
+^^^^^^^^^^^^^^^^^^^^
+
+The ``clean`` command can be invoked like so::
+
+
+    $ nextflow clean [options]
+
+
+Extended description
+^^^^^^^^^^^^^^^^^^^^
+Upon invocation within a directory, ``Nextflow`` creates a project specific ``.nextflow.log`` file, ``.nextflow`` cache directory as well as a ``work`` directory. The ``clean`` option is designed to facilitate rapid iteration without the clutter introduced by previous executions.
+
+
+
+Options
+^^^^^^^^^^^^^^^^^^^^
+
++---------------------------+------------+--------------------------------------------------------------------------------+
+| Name, shorthand (if any)  | Default    | Description                                                                    | 
++===========================+============+================================================================================+
+| -after                    |     -      | Clean up runs executed *after* the specified one.                              |
++---------------------------+------------+--------------------------------------------------------------------------------+
+| -before                   |     -      | Clean up runs executed *before* the specified one.                             |
++---------------------------+------------+--------------------------------------------------------------------------------+
+| -but                      |     -      | Clean up all runs *except* the specified one.                                  |
++---------------------------+------------+--------------------------------------------------------------------------------+
+| -dry-run, -n              |   false    | Print names of files to be removed without deleting them.                      | 
++---------------------------+------------+--------------------------------------------------------------------------------+
+| -force, -f                |   false    | Force clean command.                                                           |
++---------------------------+------------+--------------------------------------------------------------------------------+
+| -help, -h                 |   false    | Print the command usage.                                                       |
++---------------------------+------------+--------------------------------------------------------------------------------+
+| -keep-logs, -k            |   false    | Removes only temporary files but retains execution log entries and metadata.   |                                           
++---------------------------+------------+--------------------------------------------------------------------------------+
+| -quiet, -q                |   false    | Do not print names of files removed.                                           |
++---------------------------+------------+--------------------------------------------------------------------------------+
+
+
+
+Examples
+^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
Please let me know if we're open to adding the Asciinema `Shell casts` in `Usage` section. 

These GIFs can improve the beginner friendliness of the docs and this way we can cover the scenario and provide more context for any command.

[![asciicast](https://asciinema.org/a/363719.svg)](https://asciinema.org/a/363719)

Reply:
Let's leave this out for now.

Reply:
Okey dokey 👍